libs/platform/kernel/registry_helper_km.c

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#include "ebpf_registry_helper.h"
+
+void
+close_registry_key(ebpf_registry_key_t key)
+{
+    ZwClose(key);
+}
+
+_Success_(return == 0) NTSTATUS
```

Comment:
```suggestion
NTSTATUS
```

Reply:
I filed issue #2123, which captures addressing this as well as other hardening specific to the existing registry APIs.